mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/MockIdentitySystemApplication.java

```diff
@@ -0,0 +1,12 @@
+package io.mosip.esignet.mock.identitysystem;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication(scanBasePackages = { "io.mosip.mock.identity.system" })
```

Comment:
seems like the scanbase package name is incorrect @HimajaDhanyamraju2 
it should be io.mosi.esignet.mock